plugins/woocommerce-admin/client/products/layout/product-form-layout.scss

```diff
@@ -21,4 +23,48 @@
 			margin-top: $gap-largest + $gap-smaller;
 		}
 	}
+
+	.components-tab-panel__tabs {
+		position: fixed;
+		top: $adminbar-height + $header-height;
+		left: $admin-menu-width;
+		width: calc(100% - $admin-menu-width);
+		background: $white;
+		z-index: 1001;
+		border-bottom: 1px solid $gray-400;
+		border-top: 1px solid $gray-400;
+		padding: 0 var(--large-gap) 0 var(--large-gap);
+
+		@include breakpoint( '<782px' ) {
+			top: $adminbar-height-mobile + $header-height;
+			width: 100%;
+			left: 0;
+		}
+
+		@include breakpoint( '782px-960px' ) {
+			width: calc(100% - 36px);
+			left: 36px;
+		}
+	}
+
+	.components-tab-panel__tabs-item {
+		min-height: $product-form-tabs-height;
+		padding-left: 0;
+		padding-right: 0;
+		margin-right: $gap-large;
+		font-weight: 400;
+
+		&:last-child {
+			margin-right: 0;
+		}
+
+		&.is-active {
+			font-weight: 600;
+			box-shadow: inset 0 0 0 var(--wp-admin-border-width-focus) transparent, inset 0 -3px 0 0 var(--wp-admin-theme-color);
+		}
+	}
 }
+
+.woocommerce-admin-product-layout .woocommerce-layout__header {
+	min-height: $header-height + $product-form-tabs-height;
```

Comment:
This is a hacky workaround.  Ideally we would SlotFill this into the header, but there are 2 issues:

1. We don't have a slot that works well for this case/layout.
2. The `TabPanel` does not have a way to separate out just the tabs, so we will end up just copying over most of that component from Gutenberg to separate the tabs and content.

This current solution will present issues if another plugin or party modifies the header height or menu widths.  We should follow up on this to create the correct slots.